prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -196,7 +196,8 @@ for det in `echo $LIST_OF_DETECTORS | sed "s/,/ /g"`; do
 done
 PVERTEX_CONFIG="--vertexing-sources $TRACK_SOURCES --vertex-track-matching-sources $TRACK_SOURCES"
 
-has_detector_reco FT0 && PVERTEX_CONFIG+=" --validate-with-ft0"
```

Comment:
Do we want to disable this also for the FST with simulated data?
We could use `[[ $IS_SIMULATED_DATA == 1 ]] && `?

Reply:
I don't think this is needed: it is used for the cleanup of fake vertices and has a relatively small effect.